mods/default/models/torch_floor.obj

```diff
@@ -0,0 +1,48 @@
+# Blender v2.73 (sub 0) OBJ File: 'mt_torch_full_vertices3.blend'
+# www.blender.org
+mtllib mt_torch_full_vertices3.mtl
+o Cube_Cube.001
+v 0.061653 -0.496733 0.061674
+v 0.061653 -0.496733 -0.062460
+v -0.059258 -0.496733 0.061674
+v -0.059258 -0.496733 -0.062460
+v -0.059994 -0.497234 0.497315
+v -0.059994 -0.497234 -0.498102
```

Comment:
Gap seems small enough to not be noticed; but have you considered having the torch actually stick inside the node a little bit here, like with the wall-torch? Or would that be too hard for avoiding z-fighting?